### DISCUSSION TITLE
Trying to control WS2812b led strip with only 60 leds.

### DISCUSSION THREAD

**Discussion by feder-ico** [arendst/Tasmota#15980] [Q&A] [answered]
Hi Dears,
I am trying to control a 1 meter WS2812B led strip that **only has 60 leds**.
The strip works fine when I control them with a firmware I wrote based on FastLED, so I know the circuit is correct but when I flash the ESP8266 with Tasmota, only half of the strip turns on and the other half remains off.
With FastLED you can define the number of leds, among other parameters such strip model (WS2812B) and color order (GRB) but here with Tasmota I don't know if it's possible, or what I could be doing wrong.
My current configuration is as follows:

```
  #ifdef USE_WS2812_HARDWARE
    #undef  USE_WS2812_HARDWARE
    #define USE_WS2812_HARDWARE  NEO_HW_WS2812X     // Hardware type (NEO_HW_WS2812, NEO_HW_WS2812X, NEO_HW_WS2813, NEO_HW_SK6812, NEO_HW_LC8812, NEO_HW_APA106, NEO_HW_P9813)
  #endif

  #ifdef USE_WS2812_CTYPE
    #undef  USE_WS2812_CTYPE
    #define USE_WS2812_CTYPE  NEO_GRB           // Color type (NEO_RGB, NEO_GRB, NEO_BRG, NEO_RBG, NEO_RGBW, NEO_GRBW)
  #endif 
```
Any clue?

Thanks in advance.
Federico.-

**Comment by sfromis** [comment, marked as answer]
With Tasmota, you should set the number of leds too, using the `Pixels` command.

**Reply by feder-ico** [reply]
Hi @sfromis 
It worked!
I wrote `pixels 60` on the console and all the leds are working now.
many thanks!
